public/playersBetController.js

```diff
@@ -0,0 +1,140 @@
+"use strict";
+export default class playersBetController {
```

Comment:
Classes should always start with an uppercase letter, this indicates that it is a class constructor
```suggestion
export default class PlayersBetController {
```